src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -11,200 +11,252 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
 
-name: 15.2.6. [TC-OPCREDS-3.7] Add Second Fabric over CASE [DUT-Server]
+name: 
+  15.2.6. [TC-OPCREDS-3.7] Add Second Fabric over CASE [DUT-Server]
+
 
 PICS:
     - OPCREDS.S
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic Information"
+    cluster: "Operational Credentials"
     endpoint: 0
 
 tests:
-    - label: "Pre-Conditions"
+    - label: "Precondition"
       verification: |
-          TH1 and TH2 are 2 clients that trust each other
+        TH1 and TH2 are 2 clients that trust each other
       disabled: true
 
-    - label:
-          "Step 1: Factory Reset DUT (to ensure NOC list is empty at the
-          beginning of the following steps)"
+    - label: "Wait for the alpha device to be retrieved "
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Step 1: Factory Reset DUT (to ensure NOC list is empty at the beginning of the following steps)"
       PICS: OPCREDS.S.A0000
-      verification: |
-          On both DUT and TH side, on Raspi we do factory reset with the below command. The DUT for cert should follow vendor specific procedure for factory reset
-          sudo rm -rf /tmp/chip_*
+      # verification: ""
```

Comment:
This should probably document why we are not in fact doing this step... Probably because we assume we are starting with a freshly commissioned device?